### ISSUE TITLE
Not generating performance data

### ISSUE THREAD

**Issue by p-shukla** [stockholmuniversity/nagios-plugin-elasticsearch#2]
Hello,

The plugin does not generate the performance data so can not be used for nagiosgraph service.

**Comment by simmel** [comment]
That is true, is that something you want to contribute? IIRC it's not very hard to add, just one line.
